.github/workflows/canister-tests.yml

```
@@ -770,6 +770,13 @@ jobs:
       - name: Release notes
         run: cat ${{ steps.prepare-release.outputs.notes-file }}
 
+      # Create a sha file for dfx pull, referenced by the metadata set in the build
+      - name: Create sha for dfx pull
+        run: |
+          shasum -a 256 \
+            ./internet_identity_dev.wasm.gz | cut -d ' ' -f1 > \
+            ./internet_identity_dev.wasm.gz.sha256
```

```suggestion
          shasum -a 256 internet_identity_dev.wasm.gz > \
            internet_identity_dev.wasm.gz.sha256
```

It is conventional to put the whole output of `shasum` into `.sha256` file.
So that users can `shasum -c  internet_identity_dev.wasm.gz.sha256` to check file integrity.

I made `dfx deps` to recognize `shasum` format so that projects providing `*.sha256` files can get out of box support.

---

Done!